example/exporter/kubelet-servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+# You need to have "--kubelet-object=kube-system/kubelet" as a prometheus-operator arg
```

Comment:
@brancz I'm removing the ServiceMonitor manifests as they are already added in the user guide "Cluster Monitoring".